monitor/monitoring-dam-generator-core/src/test/java/eu/seaclouds/monitor/monitoringdamgenerator/tests/MonitoringGenerationTest.java

```diff
@@ -28,7 +28,7 @@ public void monitoringInformationGenerationTest() throws Exception {
 
         Map<String, String> parametersTest;
         
-        MonitoringDamGenerator service = new MonitoringDamGenerator(new URL("http://127.0.0.1:8170"));
+        MonitoringDamGenerator service = new MonitoringDamGenerator(new URL("http://127.0.0.1:8170"), new URL("http://127.0.0.1:8083"));
```

Comment:
Are you starting two servers before the tests? Are they mocked or sth like this? Just to clarify, could you briefly explain me how this service is started with the local ips?